ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js

```diff
@@ -201,6 +233,7 @@ module.exports = plugin(
         federation: federation,
         sorts: sorts,
         'detail-level': detailLevel,
+        spellcheck: spellcheck,
```

Comment:
✏️ Could use shorthand. Ex:
```suggestion
        spellcheck,
```